src/stylesheets/elements/_inputs.scss

```diff
@@ -1,3 +1,21 @@
+$input-line-height: 1.3;
+$input-border-width: 1px;
+$input-padding-v: 1rem;
+$input-padding-h: 0.7em;
+
```

Comment:
yes, they are used twice -- once for the property declarations and once for the height property computation. i'm not sure i fully understand the second criteria (for example, aren't there certain variables that won't be updated yet are still valuable to have abstracted because they're used throughout?)